games-emulation/4do_libretro/4do_libretro-1.3.2.4.20180219git.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="A port of 4DO, a 3DO emulator to the libretro architecture"
+DESCRIPTION="4DO is an open-source, low-level emulator for the Panasonic 3DO, based on the FreeDO source code."
+HOMEPAGE="http://www.fourdo.com/"
+COPYRIGHT="2011-2018 the 4DO/FreeDO team, the libretro team"
+LICENSE="
+	FreeDO
+	GNU LGPL v3
+	"
+REVISION="1"
+srcGitRev="12eba56e3ddb1cd3c53bf26f62adeca7cc0389af"
+SOURCE_URI="https://github.com/libretro/4do-libretro/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="1aee495919d1cc0f71f965c1d6fc61a548cad36f1826aaf52207c0650dbf2886"
+SOURCE_FILENAME="libretro-4do-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="4do-libretro-$srcGitRev"
+ADDITIONAL_FILES="4do_libretro.info.in"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	4do_libretro$secondaryArchSuffix = $portVersion
+	lib:4do_libretro$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+SUPPLEMENTS="
+	retroarch$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	sed -e "s/@DISPLAY_VERSION@/v${portVersion%~.*}/" \
```

Comment:
Maybe this is not a good idea, but if you don't want to see the `git` suffix in the core info then:
```diff
-	sed -e "s/@DISPLAY_VERSION@/v${portVersion%~.*}/" \
+	sed -e "s/@DISPLAY_VERSION@/v${portVersion%git}/" \
```

Reply:
I like it actually.